lib/server-manager.ts

```diff
@@ -258,27 +257,10 @@ export class ServerManager {
     });
   }
 
-  public determineProjectPath(textEditor: TextEditor): string | null {
-    const filePath = textEditor.getPath();
-    if (filePath == null) {
-      return null;
-    }
-    return this._normalizedProjectPaths.find((d) => filePath.startsWith(d)) || null;
-  }
-
-  public updateNormalizedProjectPaths(): void {
-    this._normalizedProjectPaths = atom.project.getDirectories().map((d) => this.normalizePath(d.getPath()));
-  }
-
-  public normalizePath(projectPath: string): string {
-    return !projectPath.endsWith(path.sep) ? path.join(projectPath, path.sep) : projectPath;
-  }
-
```

Comment:
I think we should just move this code to auto-languageclient so the user can override them. Maybe there is a better way to make this configurable instead of moving the methods